src/CMakeLists.txt

```diff
@@ -525,7 +525,8 @@ target_link_libraries(
         PUGIXML::PUGIXML
         ZIP::ZIP
         ZLIB::ZLIB
-        ${QT_LIBS})
+        ${QT_LIBS}
+        nanobench)
```

Comment:
This is not a conditional include, which means even production releases are linked against the library. Do we want that? Or am I missing something?

Reply:
Yeah, even production builds would be. Maybe we should set it to debug only? but then you don't want to be benchmarking debug builds...

Maybe I should add // PLACEHOLDER like SlySven does to solve this.

Reply:
I'm wondering, whether a CMAKE option to include the benchmarking bits would be better, including a custom wrapper for the benchmark code, that simply does nothing if compiled without benchmark flag... Thinking out loud, I don't want to blow this out of proportion.